web/frontend/tsconfig.json

```diff
@@ -13,5 +13,6 @@
 
     "noPropertyAccessFromIndexSignature": false,
     "exactOptionalPropertyTypes": false,
-  }
+  },
+  "include": ["src/**/*.ts", "src/**/*.d.ts", "src/**/*.svelte"]
```

Comment:
Type-aware ESLint is going to be unhappy that this doesn't include `vite.config.ts` et. al.

Most favorite (aka "least hated") way to deal with this is to have a `tsconfig.json` that includes everything but also sets `noEmit: true` and then a separate `tsconfig.build.json` that has export options and does _not_ include non-src files

Reply:
Oh nice! This solution makes sense. I've updated this PR to more closely resemble the sdk repo